etcd/fs/fs_test.go

```diff
@@ -55,25 +91,15 @@ func TestFs1(t *testing.T) {
 		Metadata:   superblock,
 		DataPrefix: etcdfs.DefaultDataPrefix,
 	}
-	//var etcdFs afero.Fs = NewEtcdFs()
```

Comment:
Why'd you nuke this and below?

Reply:
I assumed that var was stale, I can put it back, but `mkdir /` doesn't make much sense and will always error. What's the point?